### ISSUE TITLE
Install via Carthage for Mac platform fails

### ISSUE THREAD

**Issue by skreutzberger** [stephencelis/SQLite.swift#340]
Hi,

when I run this command:
`carthage update --platform Mac --configuration Debug` 

then I get the following output error:
```
*** Building scheme "SQLite Mac" in SQLite.xcodeproj
*** WARNING: Skipping build for "macosx" SDK because the necessary signing identity "-" is not installed
```

I have more frameworks in my Carthage file and the issue just occurs with SQLite.swift.

How can I fix it?

**Comment by mikemee** [comment]
Does it work without the --configuration Debug flag?

**Comment by skreutzberger** [comment]
no, I tried both variations.

**Comment by mikemee** [comment]
What version of carthage do you have installed?  I have 0.11.0, per:
```
$ carthage version
0.11.0
```
and this seems to work ok. A quick Google suggests this might be the problem, fixed in a more recent Carthage build: https://github.com/Carthage/Carthage/issues/897. Hoping that is the issue here, because this works for me:
```
$ carthage update --platform Mac --configuration Debug
*** Fetching SQLite.swift
*** Checking out SQLite.swift at "0.9.2"
*** xcodebuild output can be found in /var/folders/fn/m4qv73410870r7xf2fpppwmm0000gn/T/carthage-xcodebuild.M3eiHQ.log
*** Building scheme "SQLite Mac" in SQLite.xcodeproj
$
```

**Comment by skreutzberger** [comment]
Perfect, I could fix it by upgrading from Carthage 0.10.0 to 0.11.0!

SQLite.swift now even installs flawlessly via Carthage if the code signing of my project is set to Mac App Store at Debug AND Release build config.

Thanks again!
